open-policy-agent/authorization/prevent_default_serviceaccount_usage/template.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8spoduniqueserviceaccountvalidation
+  annotations:
+    description: Checks wheter the pod has a unique serviceAccount, correlated with the pod's name
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sPodUniqueServiceAccountValidation
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package K8sPodUniqueServiceAccountValidation
+
+        missing(obj, field) = true {
+          not obj[field]
+        }
+        
+        missing(obj, field) = true {
+          obj[field] == ""
+        }
+
+        violation[{"msg": msg}] {
```

Comment:
@tommeramber @michaelkotelnikov 
This violation is irrelevant because when pod without serviceAccountName is created in Openshift the field is added automatically before initiating the creation request of the object. 
Therefore, there is no such case in which serviceAccountName is empty.